### PR TITLE
feat: update prompt for app develop

### PR DESCRIPTION
**Pull request by adldotori**
* generate a playground, the place where eval stores its code.
* can execute eval without a GPU
* delete python-repl (replace CodeEditor + Terminal)
* simplify EVAL prompt

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -11,7 +11,9 @@ services:
       - ../.cache/huggingface/:/root/.cache/huggingface/
       - ./static/:/app/static/
     ports:
-      - "8000:8000"
+      - "7500:7500"
+      - "7501:7501"
```
Comment (hanchchch) [inline docker-compose.yml:15]: what are these for

File: api/main.py [hunk 2]
```diff
@@ -37,21 +38,27 @@
...
+        ExitConversation(),
+    ]
+    + [
+        Text2Image("cuda"),
+        ImageEditing("cuda"),
+        InstructPix2Pix("cuda"),
+        VisualQuestionAnswering("cuda"),
+    ]
+    if torch.cuda.is_available()
+    else []
```
Comment (hanchchch) [inline api/main.py:55]: good

File: core/tools/cpu.py [hunk 3]
```diff
@@ -69,35 +73,44 @@ def read(self, inputs: str) -> str:
...
+            f"Output Answer: {output}"
         )
         return output
 
     @tool(
         name="CodeEditor.WRITE",
         description="Write code to create a new tool. "
         "If the code is completed, use the Terminal tool to execute it, if not, append the code through the CodeEditor.APPEND tool. "
-        "Input should be filename and code. "
+        "Input should be filename and code. Filename should be start with './playground/'. "
```
Comment (hanchchch) [inline core/tools/cpu.py:97]: what if it trys to write `./playground/..` ? shouldn't it have hardly coded rules not only prompt?

such as: 
```python
if not str(Path("filepath").resolve()).startswith(Path(".").resolve()):
    raise Exception("nah")
```

Comment (adldotori) [reply core/tools/cpu.py:97]: I agree. I didn't think of that.

File: docker-compose.yml (hunk 1 above)
Comment (adldotori) [reply docker-compose.yml:15]: port in docker for serving frontend, backend app

Comment (adldotori) [reply docker-compose.yml:15]: I need to give an open port to EVAL, but I'm wondering how to give it. 1. order to find the open port directly, 2. tell the open port at the system prompt, 3. tell the user prompt.

**Pull request merged**